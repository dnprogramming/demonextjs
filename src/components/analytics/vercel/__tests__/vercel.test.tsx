import { render } from "@testing-library/react";
import Vercel from "../vercel";
import React from "react";

describe("Vercel Component", () => {
    it("renders vercel analytics component unchanged", () => {
        const { container } = render(<Vercel />);
        expect(container).toMatchSnapshot();
    });
});