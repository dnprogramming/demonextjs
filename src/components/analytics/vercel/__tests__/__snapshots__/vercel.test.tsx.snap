// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Vercel Component renders vercel analytics component unchanged 1`] = `<div />`;
